vuepress/docs/v7.0/docs/reference/databases.md

```diff
@@ -84,82 +74,89 @@ The EntandoDatabaseService custom resource looks like this:
+An example of the Secret that provides the admin credentials, identified as
+`secretName` above:
 
     Secret:
       name: string, any K8S compliant name
-      namespace: string, the namespace this will be created in
+      namespace: string, the namespace the DB is created in
       stringData:
         username: string, name of an admin user that can create schemas and other users
         password: string, password of the above user
 
-### How it works
-
-In order for the EntandoApp and EntandoPlugin deployer to pick up the
-correct database service, the EntandoDatabaseService needs to be created
-BEFORE the EntandoApps and EntandoPlugins are created. There can be
-multiple EntandoDatabaseServices in the namespace, but they need to
-point to database services of different vendors, i.e. PostgreSQL, Oracle
-and MySQL. Entando currently cannot enforce any validation but if there
-are two EntandoDatabaseServices that have the same DBMS vendor, it will
+### How It Works
+#### Database Custom Resource
+In order for the EntandoApp and Plugin deployer to choose the
+correct database service, the `EntandoDatabaseService` custom resource needs to be created
+**BEFORE** the app and plugins are created. There can be
+multiple `EntandoDatabaseServices` in the namespace, but they need to
+point to DBMS of different vendors, i.e. PostgreSQL, Oracle
+and MySQL. 
+
+Entando currently does not enforce any validation but if there
+are two `EntandoDatabaseServices` that have the same DBMS vendor, it will
 simply pick the first one and continue. Please ensure that only one
-EntandoDatabaseService exists for each DBMS vendor you need to use.
+`EntandoDatabaseService` exists for each DBMS vendor used.
 
-Once the appropriate EntandoDatabaseServices have been created, any
-EntandoApp or EntandoPlugin that is created will have to specify the
+#### Spec.dbms
+Any application or plugin that is created, now has to specify the
 appropriate DBMS vendor in their `spec.dbms` property. If the
-EntandoOperator detects an EntandoDatabaseService with a matching DBMS
+Entando Operator detects an `EntandoDatabaseService` with a matching DBMS
 vendor, it will continue to create the necessary schemas on the specific
-database. If the EntandoOperator does not detect an
-EntandoDatabaseService with a matching DBMS vendor, it will fall back
-onto its default behaviour which is to create a matching Deployment and
-spin up a database service from the same namespace. If the `spec.dbms`
-property is not specified on an EntandoApp, the EntandoOperator will
-default to PostgreSQL. If the `spec.dbms` property is not specified on
-an EntandoPlugin, the EntandoOperator will assume that the EntandoPlugin
-in question does not require a database and hence bypass any database
+database. 
+
+If the Operator does not detect an
+`EntandoDatabaseService` with a matching DBMS vendor, it will fall back
+to its default behaviour--creating a matching deployment and
```

Comment:
behavior, which is to create a matching deployment and spin up a database service in the same namespace

Reply:
why thumbs down? use of a double hyphen to avoid a complete and grammatically correct sentence is awkward ??

Reply:
it's fine to use a double hyphen for a non-dependent clause